### PR TITLE
Add a Nomad Security group

### PR DESCRIPTION
**Pull request by bascht**
Shameless copy / adaption of the `sg_consul` module. :)

Ports fetched from the [Nomad site](https://www.nomadproject.io/docs/agent/config.html).

### CODE REVIEW

File: sg_nomad/main.tf [hunk 1]
```diff
@@ -0,0 +1,76 @@
+//
+// Module: tf_aws_sg/sg_nomad
+//
+// Nomad ports from https://www.nomadproject.io/docs/agent/config.html
+//
+
+// Provider specific configs
+provider "aws" {
```
Comment (antonbabenko) [inline sg_nomad/main.tf]: provider section should not be specified in module. See #23 for details.

File: sg_nomad/main.tf [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    // allows traffic from the SG itself for udp
+    ingress {
+        from_port = 0
+        to_port = 65535
+        protocol = "udp"
+        self = true
+    }
+
+    // allow traffic for TCP 22 (SSH)
+    ingress {
```
Comment (antonbabenko) [inline sg_nomad/main.tf:30]: Is SSH port really needed for Nomad?

Comment (bascht) [inline sg_nomad/main.tf:30]: It was adapted from https://github.com/terraform-community-modules/tf_aws_sg/blob/master/sg_consul/main.tf#L29-L35 – I suppose that's a convenience if it is the only SG for that resource?

File: sg_nomad/main.tf (hunk 1 above)
Comment (bascht) [inline sg_nomad/main.tf]: Removed as of 07ee252.

File: sg_nomad/main.tf (hunk 2 above)
Comment (antonbabenko) [inline sg_nomad/main.tf:30]: Ok, make sense.

**Comment by bascht** [comment]
Thanks! :smile_cat:

**Pull request merged**